lib/puppet_x/vmware/vmware_lib/puppet/property/vmware.rb

```diff
@@ -0,0 +1,9 @@
+#
+# Wrapper to load the transport library from vmware_lib
```

Comment:
I'm not trying to take away from a lot of good work, but you have a cut-and-paste slip -- _this_ is loading the 'property library'